LinqOverWCF/Windows Host/README.md

```diff
@@ -1,12 +1,17 @@
+# How to run
+
+First of all, you need to install Northwind DB from Microsoft to your SQL Server instance.
```

Comment:
add reference to https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/sql/linq/downloading-sample-databases